docs/global/Postman.md

```diff
@@ -1,5 +1,14 @@
-Click the button to import our collection of API examples to [Postman](https://www.getpostman.com/).
+!!! tip "Postman API documentation"
 
-[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/a50bfb3e105fc3d24402)
+    ![Postman documentation](https://www.postman.com/assets/logos/pm-orange-logo-horiz.svg){style="height: 6em;"}
 
-or [download the raw JSON collection file](/postman/a50bfb3e105fc3d24402.json).
+    Navigate to [the Postman API documentation](https://api.besu.hyperledger.org/)
```

Comment:
When we say "the Postman API collection" we should probably be more clear that the user will be viewing the Besu APIs in Postman. So could we maybe update this to something like:

```suggestion
    View this [Hyperledger Besu JSON-RPC API documentation in Postman](https://api.besu.hyperledger.org/) 
    and view example requests in multiple coding languages. You can also run requests directly in Postman.
```